lab_8_llm/main.py

```diff
@@ -43,6 +46,11 @@ def obtain(self) -> None:
         Raises:
             TypeError: In case of downloaded dataset is not pd.DataFrame
         """
+        self._raw_data = (load_dataset(self._hf_name, split='train')).to_pandas()
+
+    @property
+    def raw_data(self) -> DataFrame:
```

Comment:
The method has already been declared in the parent class `AbstractRawDataImporter`

Reply:
done

Reply:
sorry, not done
without this I can't pass mypy checks

Reply:
Have the tests passed now?

Reply:
I did not see that these changes remained. Delete this method and variable and push it, let's look at the mypy error in the tests after update to last master